### DISCUSSION TITLE
💡 RFC: EditorConfig compatibility

### DISCUSSION THREAD

**Discussion by cdtut** [withastro/language-tools#902] [Feature requests]
### Background & Motivation

EditorConfig is very convenient standard for setting indent style and to trim trailing whitespace to do minimal formatting during saving. It is also standard that work in lots of editors.

Can new setting be added to disable prettier formatting? Only way to disable it is disable all formatting. But then have to spend lots of time changing indent and trim trailing whitespace in manual way because no formatting is possible.

If new setting is there then it is possible to disable prettier but EditorConfig extension can still do its useful formatting which helps a lot.

### Possible solutions

New setting to disable prettier in astro files but compatible with EditorConfig settings.

### Help make it happen!

- [ ] I am willing to submit a PR to implement this change.
- [ ] I am willing to submit a PR to implement this change, but would need some guidance.
- [ ] I am not willing to submit a PR to implement this change.

**Comment by rogama25** [comment]
A similar thing happens to me, I want to keep my indentations to 2 spaces but the Astro plugin keeps formatting it to 4 spaces. Seems to ignore the tab_width setting even though Prettier documentation explicitly says that is supported [here](https://prettier.io/docs/en/options.html#tab-width)
